userguide/services.rst

```diff
@@ -1635,7 +1627,9 @@ unless there is a specific need.**
   back up the data, create a new case-insensitive dataset, create an
   SMB share on it, then copy the data from the old one onto it. After
   the data has been checked and verified on the new share, the old one
-  can be deleted.
+  can be deleted. Once the case-insensitive dataset is created, the
```

Comment:
This seems to be in the wrong place.  Setting the parameter should be done right after creating the new share, before copying data.

Also, the "Once" construct can be confusing to non-native English speakers.  "When" or "After" is somewhat clearer.